### ISSUE TITLE
"🤔 Question"   Some confusion about VQBet

### ISSUE THREAD

**Issue by SHUNLU-1** [huggingface/lerobot#352]
First of all thank you for open sourcing very good code ,
I according to your code training vqet found a strange phenomenon,Finished discretizing action data! Why are loss and grdn so abnormal? Do you train like this
![2024-08-05 21-43-21屏幕截图](https://github.com/user-attachments/assets/cd782601-4e0e-4ff0-b045-1e5140a6306d)

There is another problem,
I trained vqbet at pusht data integration power of 50~60%. Is this the best? What are the advantages and disadvantages of vqet? 
Looking forward to your reply

**Comment by aliberts** [comment]
Hi there! Thank you for your message, and for raising this.
There's clearly an issue with your loss and gradients. What's funny is that they look fine in the top row of your screenshot.

We trained VQ-BeT on pusht data with no issues, you can have a look at our pre-trained model [on the hub](https://huggingface.co/lerobot/vqbet_pusht).
For reference, the training curves are also available [here](https://wandb.ai/jaylee0301/lerobot/runs/9r0ndphr?nw=nwuserjaylee0301).

Could you provide us with the exact command you ran for your training?
Also, can you paste here your system info? You can use this little script for this:
```bash
python lerobot/scripts/display_sys_info.py
```

cc @alexander-soare @Cadene
